tests-functional/tests/test_delete_keypair.py

```diff
@@ -0,0 +1,28 @@
+import re
+import pytest
+from clients.api import ApiResponseError
+from resources.constants import user_1, wallet_account_details_root, keypair_name
+
+
+@pytest.mark.rpc
+class TestDeleteKeypair:
+
+    @pytest.fixture(autouse=True)
+    def setup_backends(self, backend_new_profile):
+        self.account = backend_new_profile("sender")
+
+    def test_delete_keypair(self):
+        add_resp = self.account.accounts_service.add_keypair_via_private_key(
```

Comment:
The same as above, check for the keystore file existence and check after deletion that it doesn't exist.